src/NuGet.Core/NuGet.Build.Tasks.Pack/GetProjectReferencesFromAssetsFileTask.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.ProjectModel;
+
+namespace NuGet.Build.Tasks.Pack
+{
+    public class GetProjectReferencesFromAssetsFileTask : Task
```

Comment:
Add a summary explaining where this is used. The name is clear on what it does, but I find that having extra comments is helpful for msbuild tasks like this.